src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/PartnershipController.java

```diff
@@ -42,11 +44,12 @@ public ResponseEntity<Object> createPartnership(
         var transactionId = 12321123;
         var logMap = new HashMap<String, Object>();
         logMap.put(URL_PARAM_TRANSACTION_ID, transactionId);
+        Transaction transaction = new Transaction();
```

Comment:
Rather than create a new transaction object at this point you need to update your branch from 'main' and use the transaction that's passed in to the method.

Reply:
after merge with main used transaction that is passed